share/completions/irb.fish

```diff
@@ -0,0 +1,31 @@
+# irb (interactive ruby)
+
+complete -c irb -s f -d "Suppresses read of ~/.irbrc"
+complete -c irb -s d -d "Set \$DEBUG to true"
+complete -c irb -s r -f -d "Load LIB using require"
+complete -c irb -s I -ra "(__fish_complete_directories)" -d "Specify \$LOAD_PATH directory"
```

Comment:
You probably want `-x` instead of `-r` to only complete directories (not files).

Reply:
Indeed I do! Thanks for catching that.